app/src/main/java/ch/epfl/sweng/SDP/game/RankingFragment.java

```diff
@@ -1,46 +1,60 @@
 package ch.epfl.sweng.SDP.game;
 
+import android.annotation.SuppressLint;
 import android.content.Context;
+import android.content.res.AssetManager;
+import android.graphics.Bitmap;
+import android.graphics.Typeface;
 import android.os.Bundle;
 import android.support.annotation.NonNull;
 import android.support.v4.app.ListFragment;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
 import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.ImageView;
 import android.widget.TextView;
 
+import ch.epfl.sweng.SDP.R;
+import ch.epfl.sweng.SDP.auth.Account;
+import ch.epfl.sweng.SDP.firebase.Database;
+import ch.epfl.sweng.SDP.home.GameResult;
+import ch.epfl.sweng.SDP.localDatabase.LocalDbHandlerForGameResults;
+import ch.epfl.sweng.SDP.utils.SortUtils;
+
 import com.google.firebase.database.DataSnapshot;
 import com.google.firebase.database.DatabaseError;
 import com.google.firebase.database.DatabaseReference;
 import com.google.firebase.database.ValueEventListener;
 
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.HashMap;
 import java.util.List;
-import java.util.Locale;
 import java.util.Map;
 
-import ch.epfl.sweng.SDP.R;
-import ch.epfl.sweng.SDP.auth.Account;
-import ch.epfl.sweng.SDP.firebase.Database;
-import ch.epfl.sweng.SDP.utils.SortUtils;
-
-
 /**
  * A custom {@link ListFragment} used for displaying the final ranking at the end of the game.
  */
 public class RankingFragment extends ListFragment {
 
     private static final String TOP_ROOM_NODE_ID = "realRooms";
-
+    private static final int RANK = 10;
     private String roomID;
 
     private DatabaseReference rankingRef;
     private DatabaseReference finishedRef;
 
     private Map<String, Integer> finalRanking;
+
     private List<String> rankedUsernames;
 
+    private Bitmap[] drawings;
+    private String[] playerNames;
+
+    private Account account;
```

Comment:
Account is never initialized. You have to call Account.getInstance(context) somewhere.